Frameworks/CoreGraphics/DWriteWrapper.mm

```diff
@@ -23,13 +23,17 @@
 
 #import <unordered_map>
 #import <vector>
+#import <functional>
 
 using namespace std;
 using namespace Microsoft::WRL;
 
 static const wchar_t* TAG = L"_DWriteWrapper";
 static const wchar_t* c_defaultUserLanguage = L"en-us";
 
+// Static collection which holds all fonts registered by user programatically, through plist, etc.
+static ComPtr<IDWriteFontCollection> __UserCreatedFontCollection;
```

Comment:
This is absolutely not created in a threadsafe manner. Magic statics?

---
In reply to: [89407080](https://github.com/Microsoft/WinObjC/pull/1454#discussion_r89407080) [](ancestors = 89407080,89407071)